hack/common.sh

```diff
@@ -91,7 +91,7 @@ cleanup_WMCO() {
 # returns the operator version in `Version+GitHash` format
 get_version() {
   OPERATOR_VERSION=0.0.1
-  GIT_COMMIT=$(git rev-parse --short HEAD)
+  GIT_COMMIT=$(git rev-parse HEAD)
```

Comment:
Why this change?

Reply:
Is this because of `The version format was changed to full hash instead of shortened hash to
remove possibility of collisions and allow for better validation.`? I am -1 on having the full hash added to the version. We get reasonable collision prevnetion and validation with the short hash and for version that is good enough.

Reply:
from the commit message:

"The version format was changed to full hash instead of shortened hash to
remove possibility of collisions and allow for better validation."